data/testing_commits_testfile.json

```diff
@@ -0,0 +1,3817 @@
+{
+    "username": "test_data",
+    "RAW_DATA": [
+        {
+            "hash": "0fe6ab833a8a3ad046a9c802528d218c5589f4b0",
+            "author_msg": "Adding the first version of the gitignore file.",
+            "author_name": "Gregory M. Kapfhammer",
+            "author_email": "gkapfham@allegheny.edu",
+            "merge": false,
+            "line_added": 225,
+            "line_removed": 0,
+            "lines_of_code": 175,
+            "complexity": 0,
+            "methods": [],
+            "filename": [
+                ".gitignore"
+            ],
```

Comment:
This is the full commit raw data from the cogitate tool repository, we will be using a different json file to store raw data and the data should be calculated rather than added to the master branch. If you are using it for testing, I suggest that you use a smaller sample rather than the full one to avoid having a large file. Additionally, we have been considering the usage of  `tempfile` to create temporary json files for testing purposes.

Reply:
I will remove this.